scripts/ci/deploy/config/databend-query-node-1.toml

```diff
@@ -45,7 +45,7 @@ table_disk_cache_mb_size = 10240
 [log]
 level = "ERROR"
 dir = "./.databend/logs"
-query_enabled = false
+query_enabled = true
```

Comment:
Why enable `query_enabled` only for `databend-meta-node-1.toml`?

Reply:
because the local `make run-debug` uses this `databend-query-node-1.toml`